docs/root/configuration/overview/v2_overview.rst

```diff
@@ -510,6 +510,21 @@ to
 with the effect that the LDS stream will be directed to *some_ads_cluster* over
 the shared ADS channel.
 
+.. _config_overview_v2_mgmt_con_issues:
+
+Management Server Unreachability
+--------------------------------
+
+Some times because of network related issues, Envoy instance may loose connectivity with the 
```

Comment:
I would drop the first expression before `,`; there are multiple reasons we can lose connectivity. I would just start this with `An Envoy instance may loose...`.

Reply:
I have combined both the sentences like this
When Envoy instance looses connectivity with the management server, Envoy will latch on to
the previous configuration while actively retrying in the background to reestablish the 
connection with the management server.  as 
"An Envoy instance may loose connectivity with the management server." looks to end abruptly.